Server/Pages/ServerConfig.razor

```diff
@@ -134,6 +134,13 @@
                 <br />
                 <ValidationMessage For="() => Input.DBProvider" />
             </div>
+            <div class="form-group">
+                <label>Enable Remote Control Recording</label>
+                <br />
```

Comment:
minor:  `Nothing worth refactoring, but <br />` isn't semantic and shouldn't be used to control spacing between html elements. Using margin/padding is the preferred approach.

Reply:
Noted.  In this case, it's being used as an actual line break between inline elements, so margin/padding wouldn't help here.  We'd have to make the label block and wrap the checkbox in a div.